recipes/traceback-with-variables/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "traceback-with-variables" %}
+{% set version = "1.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/b5/a7/50e61fccfc2c5335d6e0fb71cfc3f41e89a82432f859fdc5da70bc4cb54c/traceback-with-variables-1.1.7.1.tar.gz
+  sha256: 59d38f74fb53dd6ac94eba2f57acc8346e140f1b55d4db4bef29d5a7f356b922
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - 'traceback-with-variables = traceback_with_variables.main:main'
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+
+test:
+  commands: 
+    - {{ name }} --help
+  imports:
+    - traceback_with_variables
+
+about:
+  home: https://github.com/andy-landy/traceback_with_variables/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Add a note that the license file is manually packaged. I opened a PR to fix this in https://github.com/andy-landy/traceback_with_variables/pull/11

```yaml
# License file manually packaged. See https://github.com/andy-landy/traceback_with_variables/pull/11
license_file: LICENSE
```

This will be a visual reminder to check whether the file in included in a future release at which point the comment and the manually packaged copy can be removed.

Reply:
Thanks!